src/landing.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
```

Comment:
Do we even need react and typescript for this page?

I guess part of it would depend on how complex the `handleUpload` will be. Did you hear back on what it should look like? Could it be just a redirect to a URL inside the jupyterlite app where query arguments would trigger upload dialog?

Reply:
I'll ask about `handleUpload` internally; I thought that `tsx` should be the way to go, but a much simpler `landing.html` is also fine with me. Currently, it is coded to open a new, untitled notebook in the tracker.